cmd/vcfFilter/vcfFilter.go

```diff
@@ -102,5 +285,7 @@ func main() {
 	infile := flag.Arg(0)
 	outfile := flag.Arg(1)
 
-	vcfFilter(infile, outfile, *groupFile, *chrom, *minPos, *maxPos, *ref, *alt, *minQual, *biAllelicOnly, *substitutionsOnly, *segregatingSitesOnly, *removeNoAncestor, *onlyPolarizableAncestors)
+	total, removed := vcfFilter(infile, outfile, c, *groupFile, parseFormat, parseInfo)
+	log.Printf("Processed  %d variants\n", total)
```

Comment:
Are these debug prints? Should be removed?

Reply:
I found myself constantly calculating this with `wc -l` after each run so I figured it would be nice to have this summary after it finished running. If that's not the style we are going for in gonomics then I'll go ahead and remove them, but I think it could be useful for a number of commands. Also, it prints to  stderr so it shouldn't cause problems with piping in unix.

Reply:
ok the piping was my only major concern. Going to approve

Reply:
Maybe a cool option would be to have a "verbose" mode/flag that would print these stats?